x-pack/plugins/elastic_assistant/server/routes/user_conversations/update_route.ts

```diff
@@ -45,23 +45,21 @@ export const updateConversationRoute = (router: ElasticAssistantPluginRouter) =>
         const { id } = request.params;
         try {
           const ctx = await context.resolve(['core', 'elasticAssistant', 'licensing']);
-          const license = ctx.licensing.license;
-          if (!hasAIAssistantLicense(license)) {
-            return response.forbidden({
-              body: {
-                message: UPGRADE_LICENSE_MESSAGE,
-              },
-            });
+          const authenticatedUser = ctx.elasticAssistant.getCurrentUser();
+          // Perform license, authenticated user and FF checks
+          const checkResponse = performChecks({
+            authenticatedUser: true,
+            capability: 'assistantKnowledgeBaseByDefault',
```

Comment:
We'll want to remove `capability` as the conversations API isn't behind a feature flag:

```suggestion
          // Perform license and authenticated user checks
          const checkResponse = performChecks({
            authenticatedUser: true,
```

Reply:
@YulNaumenko should this FF check be removed on `x-pack/plugins/elastic_assistant/server/routes/chat/chat_complete_route.ts` too?